modules/browsiRtdProvider.js

```diff
@@ -125,35 +246,78 @@ function waitForData(callback) {
   }
 }
 
+function setBrowsiTag(data) {
+  window.browsitag = window.browsitag || {};
+  window.browsitag.rtd = {
+    pvid: PVID || data.pvid,
+    d: data.d,
+    g: data.g,
+    aid: data.aid,
+    es: data.es,
+    sk: _moduleParams.siteKey,
+    pk: _moduleParams.pubKey,
+    t: TIMESTAMP
+  };
+  if (data.eap) {
+    window.browsitag.data = window.browsitag.data || {};
+    window.browsitag.data.get = getBrowsiTagRTD;
+    window.browsitag.apiReady = true;
+  }
+}
+
+function getBrowsiTagRTD(identifier) {
+  const uc = identifier || 'placeholder';
+  const rtd = getRTD([uc]);
+  return rtd[uc];
+}
+
 export function setData(data) {
   _browsiData = data;
+  if (!PVID) { PVID = data.pvid; }
   if (isFn(_dataReadyCallback)) {
     _dataReadyCallback();
     _dataReadyCallback = null;
   }
 }
 
+function getOnPageData(auc) {
+  try {
+    const dataMap = {};
+    if (window.browsitag?.apiReady) {
```

Comment:
it will be ready because you already marked it as ready
but I'm not sure why we need this